docs/user-guide/src/contributing/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to Metal³
+
+This section is dedicated to the various ways you can contribute to Metal³.
+There are many things outside of normal code contributions that you can do.
+Here are some examples:
+
+- Report issues when you find them
```

Comment:
This is so light on details, that I wonder if it makes sense to include this document. Realistically, all these bullet points require a guidance on how to do them.